dev/integration_tests/flutter_gallery/android/app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,7 @@ found in the LICENSE file. -->
     -->
     <uses-permission android:name="android.permission.INTERNET"/>
 
-    <application android:label="Gallery" android:icon="@mipmap/ic_launcher">
+    <application android:label="Gallery" android:icon="@mipmap/ic_launcher" android:name="androidx.multidex.MultiDexApplication">
```

Comment:
Changing the `android:name="${applicationName}"' will allow the tool to automatically add multidex support.

Reply:
Once the android:name is set to the above value, run a build with flutter tool and accept the multidex recommendation.